interpolated-defaults.tf

```diff
@@ -203,3 +231,11 @@ data "azurerm_private_dns_zone" "cftptl" {
   name                = each.key
   resource_group_name = local.cftptl_core_infra_resource_group
 }
+
+module "ctags" {
+  source = "github.com/hmcts/terraform-module-common-tags"
+
+  builtFrom   = var.builtFrom
+  environment = var.env
+  product     = var.product
+}
```

Comment:
Can you just use the `var.common_tags` variable instead of referencing this module please?